templates/states.yml

```
@@ -54,3 +54,10 @@ states:
     aliases: [no_aplicar, no aplicar, skip, monitor]
     description: Doesn't fit, don't apply
     dashboard_group: skip
+
+  - id: hired
+    label: Hired
+    aliases: [contratado, contratada, hired, accepted, accept]
+    description: Offer accepted, job landed!
+    dashboard_group: hired
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)